tests/__init__.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
```

Comment:
Hm, is there a benefit to keeping this file if the fixtures are in `conftest.py`?

Reply:
:man_shrugging: I think either way is fine. `conftest.py` is meant for fixtures (https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions), but as far as I know there's no reason that things that need explicit importing can't go there too.

Reply:
I think I hit issues with having a `tests/__init__.py` elsewhere. If we're not hitting any issues here, then I'm fine with keeping `tests/__init__.py`.